example_plugins/keyboard/src/lib.rs

```diff
@@ -23,7 +24,7 @@ pub struct MoveCommand {
 }
 
 /// Component identifing the cube
-#[derive(Serialize, Deserialize, Default, Clone, Copy)]
+#[derive(ComponentDerive, Serialize, Deserialize, Default, Clone, Copy)]
```

Comment:
Shouldn't we do this everywhere else we implement `Component`? Right now when I do `rg 'impl Component'` I see a number of other places this can be applied!

Reply:
Yes we should, currently I decided to have it in keyboard so that we should decide on this. As I was just testing the macro.

Reply:
I do agree that we should make the plugins use the derive macros however I feel that that's out of the scope of this PR since it's just about creating the macros, not refactoring code to use them.